src/wp-admin/includes/update-core.php

```diff
@@ -1342,16 +1342,20 @@ function update_core( $from, $to ) {
  *
  * @global WP_Filesystem_Base $wp_filesystem
  *
- * @param string   $from      Source directory.
- * @param string   $to        Destination directory.
- * @param string[] $skip_list Array of files/folders to skip copying.
+ * @param string $from      Source directory.
+ * @param string $to        Destination directory.
+ * @param array  $skip_list Array of files/folders to skip copying.
```

Comment:
Same question here: Isn't this still an array of strings, i.e. `string[]` representing the path to the files/folders to skip?

Reply:
Same answer 😉

Reply:
https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.param.pkg.html

> The datatype should be a valid PHP type (int, string, bool, etc), a class name for the type of object, or simply "mixed".